Modules/IO/TransformMatlab/include/itkMatlabTransformIO.hxx

```diff
@@ -18,138 +18,5 @@
 #ifndef itkMatlabTransformIO_hxx
```

Comment:
Why is this file not completely removed?

Reply:
mistake.  It should be completely removed.